packages/cosmo/package.py

```diff
@@ -141,7 +150,7 @@ def edit(self, spec, prefix):
                 optionsfilter.filter('NETCDFL *=.*', 'NETCDFL = -L$(NETCDF_DIR)/lib -lnetcdff -lnetcdf')
             if '+eccodes' in spec:
               optionsfilter.filter('GRIBAPIL *=.*', 'GRIBAPIL = -L$(GRIBAPI_DIR)/lib -leccodes_f90 -leccodes -L$(JASPER_DIR)/lib -ljasper')
-            makefile.filter('/Options', '/' + OptionsFileName)
+            makefile.filter('/Options.*', '/' + OptionsFileName)
```

Comment:
This is an important bug fix. Upon failure the build can be executed multiple times, and here we end up with a repeated pattern Options.tsa.pgi.tsa.pgi....

Reply:
Very nice, was wondering how to do that, thanks! I was always reseting the Makefile in between two build which was for sure not very practical.